import_odbc/import_odbc.py

```diff
@@ -35,25 +35,38 @@ class import_odbc_dbtable(orm.Model):
-        'dbsource_id': fields.many2one('base.external.dbsource', 'Database source', required=True),
-        'sql_source': fields.text('SQL', required=True, help='Column names must be valid "import_data" columns.'),
+        'dbsource_id': fields.many2one('base.external.dbsource',
+                                       'Database source', required=True),
+        'sql_source': fields.text('SQL', required=True,
+                                  help='Column names must be valid \
+                                  "import_data" columns.'),
         'model_target': fields.many2one('ir.model', 'Target object'),
-        'noupdate': fields.boolean('No updates', help="Only create new records; disable updates to existing records."),
-        'exec_order': fields.integer('Execution order', help="Defines the order to perform the import"),
+        'noupdate': fields.boolean('No updates',
+                                   help="Only create new records;\
+                                   disable updates to existing records."),
+        'exec_order': fields.integer('Execution order',
+                                     help="Defines the order to perform \
+                                     the import"),
         'last_sync': fields.datetime('Last sync date',
-                                     help="Datetime for the last succesfull sync."
-                                     "\nLater changes on the source may not be replicated on the destination"),
+                                     help="Datetime for the last succesfull \
+                                     sync. \nLater changes on the source may \
+                                     not be replicated on the destination"),
         'start_run': fields.datetime('Time started', readonly=True),
         'last_run': fields.datetime('Time ended', readonly=True),
-        'last_record_count': fields.integer('Last record count', readonly=True),
+        'last_record_count': fields.integer('Last record count',
+                                            readonly=True),
         'last_error_count': fields.integer('Last error count', readonly=True),
         'last_warn_count': fields.integer('Last warning count', readonly=True),
         'last_log': fields.text('Last run log', readonly=True),
         'ignore_rel_errors': fields.boolean('Ignore relationship errors',
-                                            help="On error try to reimport rows ignoring relationships."),
+                                            help="On error try to reimport \
+                                            rows ignoring relationships."),
         'raise_import_errors': fields.boolean('Raise import errors',
-                                              help="Import errors not handled, intended for debugging purposes."
-                                              "\nAlso forces debug messages to be written to the server log."),
+                                              help="Import errors not \

```

Comment:
Agree with @veloutin 

Also, after a `(` you can start a new indented line, instead of aligning the text at the far right. For example:

     'raise_import_errors': fields.boolean(
        'Raise import errors',
        help="Import errors not handled, intended for debugging purposes."
                "\nAlso forces debug messages to be written to the server log."),